executor/src/witgen/machines/fixed_lookup_machine.rs

```diff
@@ -185,7 +180,7 @@ pub struct FixedLookup<'a, T: FieldElement> {
     indices: IndexedColumns<T>,
     connecting_identities: BTreeMap<u64, &'a Identity<T>>,
     fixed_data: &'a FixedData<'a, T>,
-    multiplicities: BTreeMap<u64, BTreeMap<usize, T>>,
+    multiplicities: BTreeMap<u64, Vec<T>>,
```

Comment:
I already forgot what the key is, can you add it?

Reply:
The key is `identity_id`. Do you want me to add as a comment?

Reply:
Okay